client/main.lua

```diff
@@ -559,3 +530,17 @@ function MakePedFlee(ped)
     SetPedFleeAttributes(ped, 0, 0)
     TaskReactAndFleePed(ped, cache.ped)
 end
+
+lib.callback.register('qbx-vehiclekeys:client:GetClosestVehicleNetId', function()
+    local vehicle = GetVehicle()
+    if vehicle then
+        return NetworkGetNetworkIdFromEntity(vehicle)
+    end
+end)
+
+lib.callback.register('qbx-vehiclekeys:client:GetClosestPlayer', function()
```

Comment:
Same thing here as my comment on the previous PR. camelCase and _ over -.
`qbx_vehiclekeys:client:getClosestVehicleNetId` instead of `qbx-vehiclekeys:client:GetClosestVehicleNetId`